src/ext/error_marker.js

```diff
@@ -122,7 +122,12 @@ exports.showErrorMarker = function(editor, dir) {
     
     w.el.className = "error_widget_wrapper";
     el.className = "error_widget " + gutterAnno.className;
-    el.innerHTML = gutterAnno.text.join("<br>");
+    gutterAnno.text.forEach(function (annoTextLine, i) {
```

Comment:
Now that #5602 has been merged you should be able to use `gutterAnno.displayText` here to avoid issues with the text being escaped twice and being rendered incorrectly.

Reply:
Done.
I also adapted line 101.